tests/conftest.py

```diff
@@ -397,3 +398,48 @@ def minicircle_grainstats(
 def minicircle_grainstats_20220526() -> pd.DataFrame:
     """Statistics for minicircle for comparison."""
     return pd.read_csv(RESOURCES / "minicircle_grainstats_20220526.csv", index_col=0)
+
+
+# Derive fixtures for DNA Tracing
+GRAINS = np.array(
```

Comment:
Should this be a fixture itself? Maybe not.

Reply:
No, `@pytest.fixture` apply to functions, whilst `GRAINS` is a constant hence it being upper case.